src/webi-packages/utils.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+install_webi() {
+  local USERNAME="$1"
+  local PACKAGES="$2"
+
+  check_command curl
+
+  su - "${USERNAME}" <<EOF
+    if ! command -v webi &>/dev/null; then
+        echo "Installing webinstall.dev for user ${USERNAME}..."
+        curl -sS https://webi.sh/webi | sh
+        source ~/.config/envman/PATH.env
+        echo "Webinstall.dev installation completed for user ${USERNAME}"
+    fi
+
+    if [ -n "$PACKAGES" ]; then
+      echo "Installing specified packages: ${PACKAGES}"
+      webi $(echo "$PACKAGES" | tr ',' ' ')
+    fi
+EOF
+}
+
+determine_user() {
+  local USERNAME="${1:-"auto"}"
+  if [ "${USERNAME}" = "auto" ] || [ "${USERNAME}" = "automatic" ]; then
+    USERNAME=""
+    check_command awk >/dev/null 2>&1
+    POSSIBLE_USERS=("vscode" "node" "codespace" "$(awk -v val=1000 -F ":" '$3==val{print $1}' /etc/passwd)")
```

Comment:
Could you put `"$(awk -v val=1000 -F ":" '$3==val{print $1}' /etc/passwd)"` into a variable (or leave a comment)?

Maybe something like this?

```bash
# Get username of UID 1000
USERNAME_UID_1000="$(awk -v val=1000 -F ":" '$3==val{print $1}' /etc/passwd)"
POSSIBLE_USERS=("vscode" "node" "codespace" "${USERNAME_UID_1000}")
```